src/ansible_sign/cli.py

```diff
@@ -94,51 +91,36 @@ def parse_args(args):
     # command: gpg-verify
     cmd_gpg_verify = project_commands.add_parser(
         "gpg-verify",
-        help=(
-            "Perform signature validation AND checksum verification on the "
-            "checksum manifest"
-        ),
+        help=("Perform signature validation AND checksum verification on the " "checksum manifest"),
```

Comment:
```suggestion
        help=("Perform signature validation AND checksum verification on the checksum manifest"),
```

there are quite a few of these similar changes below from dropping the line break.

Reply:
🦅 👀